thunder/benchmarks/benchmark_inference.py

```diff
@@ -674,7 +754,16 @@ def parse_args() -> argparse.Namespace:
         help="Specify the folder for thunderfx_benchmark_report.",
     )
 
-    parser.add_argument("--enable-nvfp4", action="store_true", help="Enable NVFP4 quantization for linear layers")
+    parser.add_argument(
+        "--enable-nvfp4",
+        action="store_true",
+        help="Enable NVFP4 quantization for MoE GroupedSwiGLU layers (has nvfuser grouped_mm support)",
+    )
+    parser.add_argument(
+        "--quantize-linear",
+        action="store_true",
+        help="[Experimental] Quantize nn.Linear to NVFP4. Note: nvfuser has not yet implemented nvfp4_matmul translator",
```

Comment:
I'm getting a hang with `--quantize-linear`

Reply:
let's remove it